src/libtest/lib.rs

```diff
@@ -34,10 +34,7 @@
 #![deny(warnings)]
 
 #![feature(asm)]
-#![feature(libc)]
-#![feature(rustc_private)]
 #![feature(set_stdio)]
-#![feature(staged_api)]
 #![feature(panic_unwind)]
```

Comment:
This one is weird. Is this like a compile-time check whether unwinding is active?